.github/workflows/ci.yml

```diff
@@ -20,6 +20,10 @@ on:
 permissions:
   contents: read
 
+env:
+  OVERRIDE_FEATURES: 'DEPRECATE_TEMPLATE_ACTION'
```

Comment:
This isn't right; further answer in a bit

Reply:
don't we want the deprecation active in 5.9 (alpha) though?

Reply:
No, feature flags are opt-in in alpha and canary builds.

Reply:
so I should undo all the test changes? (would make doing my other deprecation easier / not be blocked on this PR)

Reply:
I hope to unblock this PR today, fwiw.

Reply:
yay!